authentication/http.go

```diff
@@ -93,31 +93,31 @@ func GetGroups(ctx context.Context) ([]string, bool) {
 	return groups, ok
 }
 
+// MiddlewareFn is function type able to return
```

Comment:
```suggestion
// MiddlewareFunc is function type able to return
```

Not a blocker, but this would follow std lib things like e.g `http.HandlerFunc`